crab/crab_cfg_ggOnly.py

```diff
@@ -29,7 +29,7 @@
 config.JobType.psetName = 'PSet.py'
 config.JobType.scriptExe = 'crab_script_ggOnly.sh'
 config.JobType.inputFiles = ['crab_script_ggOnly.py', '../scripts/haddnano.py', 'keep_and_drop.txt']
-config.JobType.sendPythonFolder = True
+#config.JobType.sendPythonFolder = True
```

Comment:
Let's delete it altogether, since this is deprecated and caused errors for me as well.

Reply:
Done